Algorithm/QCAlgorithm.cs

```diff
@@ -159,6 +159,22 @@ public QCAlgorithm()
             OptionChainProvider = new EmptyOptionChainProvider();
             FutureChainProvider = new EmptyFutureChainProvider();
         }
+        
+        /// <summary>
+        /// Set base account currency
+        /// </summary>
+        protected void SetAccountCurrency(string accountCurrency)
+        {
+            CashBook.AccountCurrency = accountCurrency;
+        }
+
+        /// <summary>
+        /// Set base account currency
+        /// </summary>
+        protected void SetAccountCurrency(string accountCurrency)
+        {
+            Portfolio.CashBook.AccountCurrency = accountCurrency;
```

Comment:
Due to temporal issues in initialization, we might need to initialize this _before_ the algorithm's initialize is called. Otherwise some items may be initialized with the default account currency, and then the account currency could be changed causing other items to be initialized with the newly configured account currency.